src/components/visualizations/chart/highcharts/plugins/test/dualAxesLabelFormatter.spec.ts

```diff
@@ -152,4 +158,46 @@ describe("dual axes label format", () => {
             expect(value).toEqual(28000);
         });
     });
+
+    describe("dualAxesLabelFormatter", () => {
+        const series: ISeriesItem[] = [];
+        const tickPositions: number[] = [1, 2, 3, 4, 5];
+        const data = {
+            axis: {
+                opposite: false,
+                series,
+                tickPositions,
+                userOptions: { defaultFormat: "" },
+                defaultLabelFormatter() {
+                    return this.value + "";
+                },
+            },
+            chart: { userOptions: { stackMeasuresToPercent: true } },
+            value: 49,
+        };
+
+        it.each([["49", "#.###"], ["4900%", "#.###%"]])(
+            "should return %s if format is %s and stackMeasuresToPercent is %s",
```

Comment:
Please check the test description, I see there are 3 `%s`, but we ony have 2 params :)

Reply:
fixed